GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,10 +8,163 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var episodes = GOTEpisode.allEpisodes
+    
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
```

Comment:
You're hard coding this value. You need to refer to the count of the data you're trying to access so if the model changes you return that count, not a number. e.g
 return arrOfSeasons.count